pkg/crawler/crawler.go

```diff
@@ -262,3 +274,36 @@ func (c *Crawler) fetchSHA1(url string) ([]byte, error) {
 	}
 	return sha1b, nil
 }
+
+func (c *Crawler) fetchPOMLicense(url string) (string, error) {
+	resp, err := c.http.Get(url)
+	if resp.StatusCode == http.StatusNotFound {
+		return "", nil
+	}
+	if err != nil {
+		return "", xerrors.Errorf("can't get pom xml from %s: %w", url, err)
+	}
+	defer resp.Body.Close()
+
+	var pomProject PomProject
+
+	decoder := xml.NewDecoder(resp.Body)
+	decoder.CharsetReader = charset.NewReaderLabel
+	err = decoder.Decode(&pomProject)
+
+	if err != nil {
+		return "", xerrors.Errorf("can't parse pom xml from %s: %w", url, err)
+	}
+
+	if len(pomProject.Licenses) == 0 {
+		return "", nil
+	}
+
+	var licenses []string
+	for _, l := range pomProject.Licenses {
+		licenses = append(licenses, l.Name)
+	}
```

Comment:
If i understand correctly from your answer( https://github.com/aquasecurity/trivy/discussions/4236#discussioncomment-6665566) - there are cases when licenses contains duplicates. 
Perhaps we need to remove duplicates here?

```suggestion
	var licenses []string
	for _, l := range pomProject.Licenses {
		licenses = append(licenses, l.Name)
	}
	
	licenses = lo.Uniq(licenses)
```

Reply:
Right. Was planning to add it. Let me commit it.

Reply:
Can you also add duplicates to the test file? To see if it works.

Reply:
Sure